Refs #33990 -- Renamed TransactionTestCase.assertQuerysetEqual() to assertQuerySetEqual(). (#16110)
django/django: docs/releases/4.2.txt

* The ``map_height`` and ``map_width`` attributes of ``BaseGeometryWidget`` are
  deprecated, use CSS to size map widgets instead.

> * Using ``assertFormsetError()`` is deprecated. Use::
> 
>     assertFormSetError(…)
> 
>   (with a capital ``S`` in ``FormSet``).

Reviewer: ```suggestion
* ``SimpleTestCase.assertFormsetError()`` is deprecated in favor of
  ``assertFormSetError()``.
```

> * Using ``assertQuerysetEqual()`` is deprecated. Use::
> 
>     assertQuerySetEqual(…)
> 
>   (with a capital ``S`` in ``FormSet``).

Reviewer: ```suggestion
* ``TransactionTestCase.assertQuerysetEqual()`` is deprecated in favor of
  ``assertQuerySetEqual()``.
```
